nixos/doc/manual/development/option-types.xml

```diff
@@ -360,6 +360,106 @@
      </para>
     </listitem>
    </varlistentry>
+   <varlistentry>
+    <term>
+     <varname>types.orderOf</varname> {
+      <replaceable>before</replaceable> ? a: b: false,
+      <replaceable>elemType</replaceable> }
```

Comment:
```suggestion
      <replaceable>wrappedType</replaceable> }
```

`elemType` would have been the entire `attrsOf foo` for example, despite the name hinting that it'd only be `elemType = foo`.